nni/contrib/compression/base/compressor.py

```diff
@@ -321,13 +321,32 @@ def get_calibration_config(self) -> Dict[str, Dict[str, Dict[str, torch.Tensor |
                         else target_space.zero_point,
                     'quant_dtype': target_space.quant_dtype if target_space.quant_dtype else 'int8',
                     'quant_scheme': target_space.quant_scheme,
+                    'quant_bits': target_space.quant_bits,
                 }
                 if target_space.tracked_max is not None:
                     calibration_config[module_name][target_name]['tracked_max'] = target_space.tracked_max.cpu()
+                elif target_space.zero_point is not None and target_space.scale is not None:
+                    tracked_max = target_space.scale * (target_space.qmax - target_space.zero_point)
+                    calibration_config[module_name][target_name]["tracked_max"] = tracked_max.cpu() \
+                        if isinstance(target_space.scale, torch.Tensor) else tracked_max
```

Comment:
a little confusing, in which case `target_space.scale` is not a tensor?

Reply:
unify `" -> '`

Reply:
done